packages/lib-sourcify/src/Validation/processFiles.ts

```diff
@@ -0,0 +1,243 @@
+// Tools to assemble SolidityMetadataContract(s) from files.
+
+import { logDebug, logInfo } from '../lib/logger';
+import { Metadata, PathBuffer, PathContent } from '../lib/types';
+import { SolidityMetadataContract } from './SolidityMetadataContract';
+import { unzipFiles } from './zipUtils';
+import fs from 'fs';
+import Path from 'path';
+/**
+ * Regular expression matching metadata nested within another string.
+ * Assumes metadata's first key is "compiler" and the last key is "version".
+ * This is true for the metadata files generated by the compiler as it's canonicalized by sorting the keys.
+ */
+const NESTED_METADATA_REGEX =
+  /"{\\"compiler\\":{\\"version\\".*?},\\"version\\":1}"/;
+const HARDHAT_OUTPUT_FORMAT_REGEX = /"hh-sol-build-info-1"/;
+
+export function createMetadataContractsFromPaths(
```

Comment:
Regarding this, I reiterate what I said yesterday, I would like the final library to be used like this:

```ts
import {Validation, Compilation, Verification} from "@ethereum-sourcify/lib-sourcify";

const metadataContracts = Validation.createMetadataContractsFromPaths(...)

const metadataContract = new Validation.SolidityMetadataContract(...)
```

What do you think? If you like it, then we can rename this file as `validation.ts` (like an index of this folder), in which we export everything related to validation:
- the functions already in this file
- and we import/export `SolidityMetadataContract`

Reply:
Ok let's go with your way, it's nice. How do we do that do we declare a `Validation` class or a namespace, or function?

Reply:
I think we can just export a default object with all the needed classes / functions in an index file

```ts
// index.ts
export default {
  Validation:{},
  Compilation:{},
  Verification:{},
}
```